pkg/contractwriter/contracwriter_facade.go

```diff
@@ -0,0 +1,29 @@
+package contractreader
+
+import (
+	"context"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+	"github.com/smartcontractkit/chainlink-common/pkg/types"
+	"math/big"
+)
+
+// ContractWriterFacade wraps the public functions of ContractWriter in chainlink-common so that we can mock it.
```

Comment:
We don't need to do this to mock it, you can just mock it directly, see how we mock the libocr packages